src/unittests/deskflow/test_KeyMap.h

```diff
@@ -0,0 +1,29 @@
+/*
+ * Deskflow -- mouse and keyboard sharing utility
+ * SPDX-FileCopyrightText: (C) 2025 Chris Rizzitello <sithlord48@gmail.com>
+ * SPDX-License-Identifier: GPL-2.0-only WITH LicenseRef-OpenSSL-Exception
+ */
+
+#include <QObject>
+#include <QTest>
+
+#include "base/Log.h"
```

Comment:
```suggestion
#include "base/Log.h"
```

#include <QObject>
#include <QTest>